wordpress-mobile/WordPress-iOS
# Homepage Settings: Add indicators and settings to pages list

This PR implements the final part of Homepage Settings:

* Indicators on cells in the Pages list to show if a site is currently the Homepage or the Posts page
* The ability to set the Homepage or Posts page by tapping the `...` menu on a row

Both of these features are only visible is the site is set to use a Static Homepage.

![homepage-1](https://user-images.githubusercontent.com/4780/84519618-40dfc700-acca-11ea-99b1-6292e0fb8e7b.gif)
![homepage-settings-1](https://user-images.githubusercontent.com/4780/84519836-8603f900-acca-11ea-865d-7c407e6f474a.png)

**To test:**

* Set a site to Static Homepage using Site Settings. Select a homepage and a posts page.
* Visit the Pages list and ensure both of these selections are reflected on the correct cells.
* Tap the `...` indicator on another cell and ensure that the "Set as homepage" and "Set as posts page" rows are visible. Tap one to set the page type.
* Tap another row and try the other option that you didn't select last time.
* Tap the `...` for the existing homepage or posts page, and ensure that you only see the option for the opposite page type. Tap the option and verify that the page type changes, and that the other page loses its status.
* Go back to Site Settings and change the homepage type back to latest posts.
* Go back to the Pages list and ensure that you don't see any Homepage or Posts Page indicators, and that you don't see the new options in the `...` menu.

PR submission checklist:
